third_party/javascript/BUILD

```diff
@@ -71,11 +77,13 @@ java_library(
         "intellij-ue-2022.2": ["@intellij_ue_2022_2//:angular"],
         "intellij-ue-2022.3": ["@intellij_ue_2022_3//:angular"],
         "intellij-ue-2023.1": ["@intellij_ue_2023_1//:angular"],
+        "intellij-ue-2023.2": ["@intellij_ue_2023_2//:angular"],
         "clion-2021.3": ["@intellij_ue_2021_3//:angular"],
         "clion-2022.1": ["@intellij_ue_2022_1//:angular"],
         "clion-2022.2": ["@intellij_ue_2022_2//:angular"],
         "clion-2022.3": ["@intellij_ue_2022_3//:angular"],
         "clion-2023.1": ["@intellij_ue_2023_1//:angular"],
+        "clion-2023.2": ["@intellij_ue_2023_1//:angular"],
```

Comment:
should be "@intellij_ue_2023_2//:angular". is it strange that this is not `@clion` ?